tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -163,6 +174,32 @@ function normalizeOptions(host: Tree, options: MigrateConvergedPkgGeneratorSchem
   };
 }
 
+function updateNpmScripts(tree: Tree, options: NormalizedSchema) {
+  updateJson(tree, options.paths.packageJson, json => {
+    delete json.scripts['update-snapshots'];
+    delete json.scripts['start-test'];
```

Comment:
do we still want the deprecation message with these commands ?

Reply:
hmm not really - user that will run the migration on his component is in charge/aware of those changes. WDYT?

Reply:
yeah if the user does that it will make sense 👍